eZ/Bundle/EzPublishCoreBundle/Command/ResizeOriginalImagesCommand.php

```diff
@@ -0,0 +1,254 @@
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesserInterface;
+
+/**
+ * This command resizes original images stored in ezimage FieldType in given ContentType using the selected variation.
+ */
+class ResizeOriginalImagesCommand extends Command
+{
+    const DEFAULT_ITERATION_COUNT = 25;
+    const DEFAULT_REPOSITORY_USER = 'admin';
+
+    /**
+     * @var \eZ\Publish\API\Repository\PermissionResolver
+     */
+    private $permissionResolver;
+
+    /**
+     * @var \eZ\Publish\API\Repository\UserService
+     */
+    private $userService;
+
+    /**
+     * @var \eZ\Publish\API\Repository\ContentTypeService
+     */
+    private $contentTypeService;
+
+    /**
+     * @var \eZ\Publish\API\Repository\SearchService
+     */
+    private $searchService;
+
+    /**
+     * @var \Liip\ImagineBundle\Imagine\Filter\FilterManager
+     */
+    private $filterManager;
+
+    /**
+     * @var \eZ\Publish\Core\IO\IOServiceInterface
+     */
+    private $ioService;
+
+    /**
+     * @var \Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesserInterface
+     */
+    private $extensionGuesser;
+
+    public function __construct(
+        PermissionResolver $permissionResolver,
+        UserService $userService,
+        ContentTypeService $contentTypeService,
+        SearchService $searchService,
+        FilterManager $filterManager,
+        IOServiceInterface $ioService,
+        ExtensionGuesserInterface $extensionGuesser
+    ) {
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+        $this->contentTypeService = $contentTypeService;
+        $this->searchService = $searchService;
+        $this->filterManager = $filterManager;
+        $this->ioService = $ioService;
+        $this->extensionGuesser = $extensionGuesser;
+
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        parent::initialize($input, $output);
+
+        $this->permissionResolver->setCurrentUserReference(
+            $this->userService->loadUserByLogin($input->getOption('user'))
+        );
+    }
+
+    protected function configure()
+    {
+        $this->setName('ezplatform:images:resize-original')->setDefinition(
+            [
+                new InputArgument('contentTypeIdentifier', InputArgument::REQUIRED,
+                    'Indentifier of ContentType which has ezimage FieldType.'),
+                new InputArgument('imageFieldIdentifier', InputArgument::REQUIRED,
+                    'Identifier of field of ezimage type.'),
+                new InputArgument('variation', InputArgument::OPTIONAL,
```

Comment:
I'd have expected those as options and not arguments.

Reply:
@bdunogier done in https://github.com/ezsystems/ezpublish-kernel/pull/2357/commits/aaf7b90621113cabc2624bc596e5acc3b1a2badd.